src/sickle.ts

```diff
@@ -268,7 +268,7 @@ class Annotator {
 
   private visitEnum(node: ts.EnumDeclaration) {
```

Comment:
visitEnum *must* skipComments to work.  I would prefer to only have the flag in the case where the caller might care to skip comments or not.